terraform/main.tf

```diff
@@ -23,34 +23,29 @@ resource "google_compute_subnetwork" "honeypot_subnet" {
   network       = google_compute_network.main.self_link
 }
 
-resource "google_compute_firewall" "containerssh-allow-all" {
-  name    = "containerssh-allow-all"
+resource "google_compute_firewall" "containerssh_allow_ssh" {
+  name    = "containerssh-allow-ssh"
   network = google_compute_network.main.self_link
 
   allow {
     protocol = "icmp"
   }
 
-  allow {
-    protocol = "udp"
-    ports    = ["0-65535"]
-  }
-
   allow {
     protocol = "tcp"
-    ports    = ["0-65535"]
+    ports    = ["22"]
```

Comment:
We also need to allow port `2333` for SSH server on gateway VM, so we can SSH into it for development,  and in the end disable port `22` from external network on sacrificial VM.

Maybe keep this `allow-all` for now. We can remove it as part of #5 so we can verify firewall rules at once

Reply:
Ah ok, then i'll recover the `allow-all` rule. I'm just afraid that if we open all ports then we will also get some attacks or something similar, that are not relevant for our Honeypot. ^^